error.md

```diff
@@ -0,0 +1,62 @@
+# Errors faced during migration with nikita v1
+
+## Installation of semver necessary
+
+When running the `bootstrap` command, I got an error stating that `semver` module was not found.
+
+### Solution
+
+npm install semver
+
+## SSH not working but no error
+
+Nikita actions are not being executed. The code exits after entering the input
+
+### Solution
+
+Worked after refactoring the code in `lib/run.coffee`
```

Comment:
again, not a solution, a quick workaround

Reply:
Yes. Having said that I have to admit that I based my approach on the tutorial and notably on the way the code is written.
Especially the part where it described how to call actions that defined in other files.

So I tried to stay consistent with it + The fact that the previous code was just not executing the actions.

Reply:
The architecture in nikita-arch to loop by module name is out of the scope of the tutorial, we shall preserve it.